scripts/.util/tools.json

```diff
@@ -1,3 +1,3 @@
 {
-  "pack": "v0.30.0-pre2"
+  "pack": "v0.30.0-rc2"
```

Comment:
Also I think pack v0.30.0 just got released so we can fix this, too.

Reply:
Updated, I assume it should be good, smoke tests run on PR should confirm.